lib/script.js

```diff
@@ -41,14 +41,14 @@ document.addEventListener("keydown", function (event) {
 });
 
 // Get the modal
-const modal2 = document.getElementById("formModal");
+const modal2 = document.getElementById("form-modal");
```

Comment:
Rename this for better code semantic 🤗 

```suggestion
const formModal = document.getElementById("form-modal");
```